pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Is this because you need to have a registry defined in pkg/master/master.go to pass to the rbacvalidation.AuthorizationRuleResolver prior to creating the Storage for the rbac objects, and there is not a way to do that with Strategy?